core-feature-pack/src/main/resources/content/bin/domain.conf.ps1

```diff
@@ -56,7 +56,7 @@ if (-Not $JAVA_OPTS) {
 }
 
 # Uncomment this to run with a security manager enabled
-# $SECMGR=$true
+# $global:SECMGR=$true
```

Comment:
Is "global:" prefix necessary here? Other variables from this file don't need "global:" prefix ...

Reply:
I'm not sure whether or not it needs to be global, but the `standalone.ps1` checks the `$global:SECMGR` parameter. So either we need to make it global or change the `standalone.ps1` to not use a global environment variable.

The following snippet is from the `standalone.ps1` script.
```
Write-Debug "sec mgr: $global:SECMGR"

if ($global:SECMGR) {
    $MODULE_OPTS +="-secmgr";
}
```

Reply:
It is set in the global scope in the common.ps1 so it should not be necessary (it is not, I've tested) to access to it with the global prefix there, I will change the standalone as well

Reply:
@tmiyargi +1, https://github.com/wildfly/wildfly-core/blob/master/core-feature-pack/src/main/resources/content/bin/common.ps1#L33